### PR TITLE
Feature/select or take video support to form

### LINKED ISSUE #51: Update the submission project UI to support selecting a video from the gallery and storing the selected video in the apps local storage.

### LINKED ISSUE #68: As a professor I would like to be able to view the video I selected as a description of a project to confirm that it is indeed the correct video.

### PR DESCRIPTION
**Pull request by asfoury**
I am unable to achieve a higher coverage because the code that has not been tested needs to be provided with a video that is on the device which is not possible to do with Cirrus.

### CODE REVIEW

**Comment by arnauds5** [comment]
When I try to add a video, the app crashes with the following error.
Does it need to be selected from a particular location ? (my video was in the download directory)

java.lang.RuntimeException: Failure delivering result ResultInfo{who=null, request=1, result=-1, data=Intent { dat=content://com.google.android.apps.photos.contentprovider/-1/2/content://media/external/video/media/39/ORIGINAL/NONE/2016643081 flg=0x1 clip={text/uri-list U:content://com.google.android.apps.photos.contentprovider/-1/2/content%3A%2F%2Fmedia%2Fexternal%2Fvideo%2Fmedia%2F39/ORIGINAL/NONE/2016643081} }} to activity {com.sdp13epfl2021.projmag/com.sdp13epfl2021.projmag.Form}: java.lang.IllegalArgumentException: Invalid column latitude

full error on https://pastebin.com/SF7rwzAw

**Review by arnauds5 — APPROVED** [review]
Looks good to me.
The media controller is not very easy to use (because if it disapears just before we click, then the button bellow are pressed).

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -1,6 +1,22 @@
 <component name="ProjectCodeStyleConfiguration">
   <code_scheme name="Project" version="173">
     <JetCodeStyleSettings>
+      <option name="PACKAGES_TO_USE_STAR_IMPORTS">
```
Comment (PaulCoral) [inline .idea/codeStyles/Project.xml:4]: `.idea/` could be ignored

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -33,6 +33,8 @@
     <string name="search">search</string>
     <string name="profile">profileButton</string>
     <string name="submit">Add a new project to the list</string>
+    <string name="play_video">Play Video</string>
+    <string name="add_video">Add Video</string>
```
Comment (PaulCoral) [inline app/src/main/res/values/strings.xml:37]: Maybe there's a need of a more precise name (maybe not)

**Review by PaulCoral — APPROVED** [review]
For me the it is fine.
However, there's two minor "issues" :

- When watching vertically filmed video, the video is not centered (maybe you should use a constraint layout).
- Maybe the video view could be invisible if no video has been loaded yet. And when a video is loaded, maybe triggering the action of the `PLAY VIDEO` button automatically

However, these are only suggestions and the core feature is there, doing the job well.

**Pull request merged**